asm2/js/script.js

```diff
@@ -0,0 +1,56 @@
+function displayForm(){
+    document.getElementById("input-form").style.display = "block";
+}
+
+function hideForm(){
+    document.getElementById("input-form").style.display = "none";
+}
+
+function takeNewData(){
+    var newData ={};
```

Comment:
Should use let instead of var (research usage of var, let, const for more)
```suggestion
    let newData ={};
```